docs/CodingGuidlines.md

```diff
@@ -0,0 +1,49 @@
+
+# Coding Conventions
+
+Keep it simple. As simple as possible, but not simpler.
+Emphasize readability. Code will be read many times, but only written once.
+
+## History of this document.
+
+* 2018/03/13 Initial version.
+
+
+## Baseline
+
+* These conventions are based on Goggle's guidlines (https://google.github.io/styleguide/cppguide.html). If nothing is stated here to the contrary, refer to the google guidelines.
+* This is a guideline, not a set of commandments. There will be exceptions. Treat them as such.
+* This document is subject to change.
+
+## Attitude
+
+There may be plenty of code violating this convention. Do not use this as justification to deviate from the style guide.
+Instead, use common courtesy and try to leave a place cleaner or at least as clean than you entered it.
+
+
+
+## Visuals
```

Comment:
We do this and have a .clang-format config file for this.

Reply:
Excellent, we'll try to come to a common one then.